lib/compute/jenkins-main-node.ts

```diff
@@ -113,8 +120,8 @@ export class JenkinsMainNode {
     }
     this.mainNodeAsg = new AutoScalingGroup(stack, 'MainNodeAsg', {
       instanceType: InstanceType.of(InstanceClass.C5, InstanceSize.XLARGE9),
-      machineImage: MachineImage.latestAmazonLinux({
-        generation: AmazonLinuxGeneration.AMAZON_LINUX_2,
+      machineImage: MachineImage.latestAmazonLinux2023({
+        cpuType: AmazonLinuxCpuType.X86_64,
```

Comment:
So cpu type is no need to be defined in previous version?

Reply:
it defaulted to x86_64, even now it is optional but good to declare it.

Reply:
Got it.